### PR TITLE
tests/refactor : try to stick to 80 letters per line

### PR DESCRIPTION
**Pull request by mrVanDalo**
I try to make scripts more readable : 

* use long options of shell commands (more descriptive)
* one option per line.

### CODE REVIEW

File: tests/emerge_ebuild.sh [hunk 1]
```diff
@@ -13,7 +13,8 @@ EBUILD="${1}"
 echo "Emerging ${EBUILD}"
 
 # Disable news messages from portage and disable rsync's output
-export FEATURES="-news" PORTAGE_RSYNC_EXTRA_OPTS="-q"
+export FEATURES="-news" \
```
Comment (simonvanderveldt) [inline tests/emerge_ebuild.sh]: this should be a single line

File: tests/emerge_ebuild.sh [hunk 2]
```diff
@@ -43,7 +44,16 @@ if [ -f "${PKG_CONF_FILE}" ]; then
...
+    --buildpkg \
+    --usepkg \
+    --onlydeps \
+    --autounmask=y \
+    --autounmask-continue=y \
+    "=${PKG_CATEGORY}/${PKG_NAME}"
 
 # Emerge the ebuild itself
-emerge -v "=${PKG_CATEGORY}/${PKG_NAME}"
+emerge \
```
Comment (simonvanderveldt) [inline tests/emerge_ebuild.sh]: this should be a single line?

File: tests/emerge_new_or_changed_ebuilds.sh [hunk 3]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
+#
```
Comment (simonvanderveldt) [inline tests/emerge_new_or_changed_ebuilds.sh]: why this superfluous whitespace?

File: tests/emerge_new_or_changed_ebuilds.sh [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
 # Determine which ebuilds are new or changed and emerge them
+
```
Comment (simonvanderveldt) [inline tests/emerge_new_or_changed_ebuilds.sh]: why this superfluous whitespace?

File: tests/emerge_ebuild.sh (hunk 1 above)
Comment (mrVanDalo) [inline tests/emerge_ebuild.sh]: for bash it is but for us humans we it is in multiple lines so we can read it better.

File: tests/emerge_new_or_changed_ebuilds.sh (hunk 3 above)
Comment (mrVanDalo) [inline tests/emerge_new_or_changed_ebuilds.sh]: It is there because of readability.

File: tests/emerge_new_or_changed_ebuilds.sh (hunk 4 above)
Comment (mrVanDalo) [inline tests/emerge_new_or_changed_ebuilds.sh]: Same here. It separates the script header from the script.

**Comment by mrVanDalo** [comment]
> Don't you think the single argument per line is a bit excessive?

I think : "Code is there to be read by other programmers".
Do you think it is worse to read, or harder to write?

I can definitively read this 
```
emerge \
    --quiet-build \
    --buildpkg \
    --usepkg \
    --onlydeps \
    --autounmask=y \
    --autounmask-continue=y \
    "=${PKG_CATEGORY}/${PKG_NAME}"
```
better than this 
```
emerge --quiet-build --buildpkg --usepkg --onlydeps --autounmask=y --autounmask-continue=y "=${PKG_CATEGORY}/${PKG_NAME}"
```
And sometimes it is not necessary to break the line to achieve the 80 chars per line rule. But I do it anyway because it improves the readability.

**Comment by mrVanDalo** [comment]
> The long form arguments I can understand even though I find them to be too verbose most of the time.

well it is good they are verbose. It helps to read the command.

**Comment by mrVanDalo** [comment]
> 80 chars per line is fine as a basic default and a good idea to fix

I don't want to make this a rule. It is just a personal rule to me. I don't know why I wrote it, sorry this should not be part of the discussion.

File: tests/emerge_new_or_changed_ebuilds.sh (hunk 3 above)
Comment (simonvanderveldt) [inline tests/emerge_new_or_changed_ebuilds.sh]: OK, I'd like to suggest to use google's shell script style/formatting guide for our shell scripts so we can ensure they are consistent.
See https://google.github.io/styleguide/shell.xml

In there they have the same format for the file header comment :)

**Comment by simonvanderveldt** [comment]
> I think : "Code is there to be read by other programmers".
> Do you think it is worse to read, or harder to write?
> 
> I can definitively read this
> 
> emerge \
>     --quiet-build \
>     --buildpkg \
>     --usepkg \
>     --onlydeps \
>     --autounmask=y \
>     --autounmask-continue=y \
>     "=${PKG_CATEGORY}/${PKG_NAME}"
> better than this
> 
> emerge --quiet-build --buildpkg --usepkg --onlydeps --autounmask=y --autounmask-continue=y "=${PKG_CATEGORY}/${PKG_NAME}"
> And sometimes it is not necessary to break the line to achieve the 80 chars per line rule. But I do it anyway because it improves the readability.

After looking at it a bit more, the split one does seem more legible, thanks for the example!

So let's use this style for lines that are longer than 80 characters (same as in [google's styleguide](https://google.github.io/styleguide/shell.xml?showone=Line_Length_and_Long_Strings#Line_Length_and_Long_Strings))

**Comment by simonvanderveldt** [comment]
The majority of the tests have been updated so they don't really have long lines anymore, closing this one

**Pull request closed without merging**